Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
Putting `venv` as a dependency of this task would be good (like in `install-hooks` or in `test`), because then that makes sure `pre-commit` is installed so that it can be run.